samples/fabric/fabric-cli/src/commands/event/unsubscribe.ts

```diff
@@ -109,29 +108,32 @@ const command: GluegunCommand = {
       throw new Error(`Error getting key and cert ${keyCertError}`)
     }
     
-    const eventMatcher = EventsManager.createEventMatcher(data.event_matcher)
-    const eventPublicationSpec = EventsManager.createEventPublicationSpec(data.event_publication_spec)
-    
-    try {
-        const response = await EventsManager.unsubscribeRemoteEvent(
-            contract,
-            eventMatcher,
-            eventPublicationSpec,
-            requestId,
-            networkName,
-            netConfig.mspId,
-            netConfig.relayEndpoint,
-            { address: data.view_address, Sign: true },
-            keyCert
-        )
-        
-        if (response.getStatus() == EventSubscriptionState.STATUS.UNSUBSCRIBED) {
-            console.log("Event Unsubscription Success for requestId:", response.getRequestId())
-        } else {
-            console.log("Unknown error")
+    const data = JSON.parse(fs.readFileSync(filepath).toString())
+    for (let i = 0 ; i < data.length ; i++) {
```

Comment:
If we want to have unsubscribe for an array of subscriptions, then we need to take input an array of request_ids for all those subscriptions.

Reply:
Done.